src/__mocks__/add-task.test.js

```diff
@@ -0,0 +1,22 @@
+import { addNewTask, tasks, inputTask } from './add-task';
+
+describe('test add element', () => {
```

Comment:
- [ ] OPTIONAL
Other _pure functions_ in your code are **not tested** aside _add item_ and _delete item_. Kindly ensure to test **all** pure functions.